sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
   and
   [PR 11810](https://github.com/Azure/azure-sdk-for-js/pull/11810)
 
+### Breaking changes
```

Comment:
I just realized that the changelog updates you made in https://github.com/Azure/azure-sdk-for-js/commit/1152ea0f9fb0d427d2df6f80da70e4271665d24b were made under the older release. Can you move those up here in this PR?